metadata.rb

```diff
@@ -3,15 +3,16 @@
 maintainer_email 'cookbooks@chef.io'
 license 'Apache-2.0'
 description 'Manage Splunk Enterprise or Splunk Universal Forwarder'
-version '1.7.3'
+long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+version '2.0.0'
 
-supports 'debian'
-supports 'ubuntu'
-supports 'redhat'
-supports 'centos'
+supports 'debian', '>= 8.9'
+supports 'ubuntu', '>= 16.04'
+supports 'redhat', '>= 6.9'
+supports 'centos', '>= 6.9'
 
 # for secrets management in setup_auth recipe
-depends 'chef-vault', '>= 1.0.4'
+depends 'chef-vault', '~> 3.1.1'
```

Comment:
This should be >=. Restrictive constraints in community cookbooks results in deps that can't be solved for a lot of people and we try to avoid that when we can.